export const handleError = message => {
    console.log(message);
}